config/locales/en.yml

```diff
@@ -4080,6 +4080,11 @@ See the %{link} to find out more about %{sitename}'s features and to start using
           from: "From"
           to: "Bill to"
           shipping: "Shipping"
+          payments_list:
+            date_time: "Date/time"
+            payment_method: "Payment method"
+            payment_state: "Payment state"
+            amount: "Amount"
```

Comment:
That's strange that this would be required only now, is it because the spec is now covering the template generation? 
It probably would have been picked up in manual testing, but I guess this shows how it's valuable to test without mocks where possible ;)